themes-default/slim/views/home.mako

```diff
@@ -13,16 +13,13 @@
 window.app = {};
 window.app = new Vue({
     store,
+    router,
     el: '#vue-wrap',
-    metaInfo: {
-        title: 'Home'
-    },
-    data() {
-        return {
-            header: 'Show List'
-        };
-    },
-    computed: Object.assign({
+    computed: {
+        isRoute() {
+            const { matched } = this.$route;
+            return matched.filter(route => route.components.default !== undefined).length >= 1;
```

Comment:
Unfortunately [I still don't really understand](https://github.com/pymedusa/Medusa/pull/4663#discussion_r202223963) why it's needed.
If you go to `/home/displayShow` it shouldn't load the home component.